app/javascript/mastodon/features/navigation_panel/index.tsx

```diff
@@ -238,25 +242,37 @@ export const NavigationPanel: React.FC = () => {
   );
 
   const bind = useDrag(
-    ({ last, offset: [ox], velocity: [vx], direction: [dx], cancel }) => {
-      if (ox < -70) {
+    ({
+      last,
+      offset: [xOffset],
+      velocity: [xVelocity],
+      direction: [xDirection],
+      cancel,
+    }) => {
+      const hasReachedDragThreshold = xOffset * xDirection > 70;
```

Comment:
This is a behavior change, isn't it? The previous behavior was cancelling if you go too far against the “rubber banding”, while right now it'd be in any direction?